administrator-guides/admin/general/README.md

```diff
@@ -0,0 +1,262 @@
+
+### Google Tag Manager Id
+
+You can use Google Tag Manager on Rocket.Chat by setting your Id on this setting.
+
+### Language
+
+This sets the default language for the server. This setting will be overridden if the user has changed his language preference.
+
+### Restart
+
+This button restarts the server.
+
+### Robots.txt File Contents
+
+This sets the content of the Robots.txt file, used to help indexing your server on different search engines and crawlers.
+
+### Show Setup Wizard
+
+For now this setting **SHOULD NOT** be changed. Setting this to `pending` and logging out can basically lock down your server. Please refer to this [issue](https://github.com/RocketChat/Rocket.Chat/issues/13840).
+
+### Site Name
+
+This sets the name of the server.
+
+### Site URL
+
+This sets the URL of the server.
+
+> Example: `https://chat.domain.com/`
+
+### Store Last Message
+
+ Store last message sent on each room. This will enable the viewing of the last message on the left sidebar (should be on by default)
+
+### Unread Count
+
+This setting changes the behavior of the unread counter
+
+- **All messages**: All messages will be counted on the counter, regardless if they are a mention or not.
+- **User mentions only**: Only mentions using `@yourUsername` will be counted on the counter.
+- **Group mentions only**: Only group mentions using `@all` or `@here` will be counted on the counter.
+- **User and group mentions only**: All mentions will be counted, this includes `@all`, `@here` and `@yourUsername`
+
+### Unread Count for Direct Messages
+
+This setting changes the behavior of the unread counter on Direct Messages
+
+- **All messages**: All messages sent by another user will be counted on the counter.
+- **Mentions only**: Only mentions using `@yourUsername` will be counted on the counter.
+
+- - -
+
+## Apps
+
+### Enable the App Framework
+
+Use this setting to enable the Rocket.Chat Apps Framework. This allows you to add Rocket.Chat Apps from our Marketplace to your server.
+
+- - -
+
+## GraphQL API
+
+Settings related to our GraphQL API. You can find more info on our GraphQL API [here](/developer-guides/graphql-api/)
+
+### GraphQL CORS
+
+Use this setting to enable [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) for the GraphQL API. Enabling CORS will make the server only accept API Calls from specific origins.
+
+> **Note:** Currently our GraphQL CORS accepts all domains. Setting this to false, will make your server only accept requests from the same domain of said server. E.g. `rocket.chat` server will only accept requests from the `rocket.chat` domain.
+
+### GraphQL Enabled
+
+This setting enables the GraphQL API on your server.
+
+### GraphQL Subscription Port
+
+Here you can choose what port will be used for subscriptions on the API. More info on subscriptions [here](/developer-guides/graphql-api/subscription/)
```

Comment:
```suggestion
Here you can choose what port will be used for subscriptions on the API. More info on subscriptions [here](/developer-guides/graphql-api/subscription/)

> **Note** Cannot currently use the same port that Rocket.Chat is running on.
```

Reply:
@MarcosSpessatto I think you can confirm this is true?

Reply:
@geekgonecrazy Yes. :slightly_smiling_face: